fastscanner/adapters/rest/scanner.py

```diff
@@ -118,3 +134,44 @@ async def websocket_realtime_scanner(
         if scanner_id:
             await service.unsubscribe_realtime(scanner_id)
             logger.info(f"Unsubscribed scanner {scanner_id}")
+
+
+@router.post("/scan")
+async def scan(
+    request: ScanRequest, service: ScannerService = Depends(get_scanner_service)
+) -> ScanResponse:
+    try:
+        ClockRegistry.set(LocalClock())
```

Comment:
This is not the place to set the clock. All the initializers need to go in the main.